creator-node/src/monitors/ipfs.js

```diff
@@ -1,5 +1,5 @@
 const { ipfs } = require('../ipfsClient')
```

Comment:
i think we can just remove this entire file? cc @dmanjunath 
and also its consumption

Reply:
that file and its monitors look safe to remove to me. can I do it in another PR so it's easier to review tho? there will be a lot more to remove!

Reply:
100% can do separately, this file will work as long as the ipfs pod is running